tests/test_app/tests/test_auth.py

```diff
@@ -0,0 +1,47 @@
+from django.conf import settings
+from django.http import JsonResponse
+
+from tests.test_app.tests.helpers import LoguxTestCase
+
+
+class LoguxAuthCommandTestCase(LoguxTestCase):
+    """ Auth command """
+
+    def setUp(self) -> None:
+        self.good_token = 'good-token'
+        self.good_user_id = '42'
+        settings.LOGUX_AUTH_FUNC = lambda user_id, token: token == self.good_token and user_id == self.good_user_id
+
+    def test_success_auth(self) -> None:
+        r: JsonResponse = self.logux_request({
+            "version": 3,
+            "secret": "secret",
+            "commands": [
+                [
```

Comment:
This is old array-like format. Now we are using objects.

Reply:
Yep, sure. It's just a test structure refactoring. I have just split up tests to different files for now. They all will be rewritten according to new proto